src/D2L.Bmx/ParameterSource.cs

```diff
@@ -0,0 +1,14 @@
+namespace D2L.Bmx;
+
+internal record ParameterSource {
```

Comment:
use a record type rather than an enum so it's:
* easy to customize `ToString()`
* easy to extend in the future to supply config file path